contracts/lib/WorkflowStructs.sol

```diff
@@ -42,4 +45,22 @@ library WorkflowStructs {
         uint256 maxMintingFee;
         uint32 maxRts;
     }
+
+    /// @notice Struct for license data for license attachment on IP registration.
+    /// @param licenseTemplate The address of the license template to be used for the licensing.
+    /// @param licenseTermsId The ID of the license terms to be used for the licensing.
+    /// @param licensingConfig The licensing configuration for the IP.
+    struct LicenseData {
+        address licenseTemplate;
+        uint256 licenseTermsId;
+        Licensing.LicensingConfig licensingConfig;
+    }
+
+    /// @notice Struct for PIL terms data for PIL registration and attachment on IP registration.
+    /// @param terms The PIL terms to be used for the licensing.
+    /// @param licenseTermsConfig The licensing configuration for the PIL terms.
+    struct PILTermsData {
+        PILTerms terms;
+        Licensing.LicensingConfig licensingConfig;
+    }
```

Comment:
PILTermsData -> licenseTermsData

Reply:
resolved in [151b15d](https://github.com/storyprotocol/protocol-periphery-v1/pull/134/commits/151b15d21f3e087653c5849979d16ec0d7e35b63)